src/lib/utils.ts

```diff
@@ -14,6 +14,14 @@ export function getNanoTimestamp() {
 
 export const pseudoRandomBytesAsync = promisify(pseudoRandomBytes);
 
+export const isValidUrl = (url: string) => {
+	try {
+		return Boolean(new URL(url));
```

Comment:
This is somewhat confusing as it implies we care about the return value of `new URL` but I think the actual expected behavior is
```suggestion
		new URL(url);
		return true;
```
as we're actually relying on `new URL` throwing for invalid URLs rather than anything to do with its return values

Reply:
@Page- I remember now why I did it like this, the linter complains about an unused rule. Shall we leave it as it is, or ignore the rule?

Reply:
I think I'd ignore the rule, with a comment as to how it's used (were relying on the throw behavior)